### PR TITLE
SAK-30691 Reset tool state in all tools during a role switch

### PR DESCRIPTION
**Pull request by ern**

### CODE REVIEW

File: portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/handlers/RoleSwitchHandler.java [hunk 1]
```diff
@@ -5,110 +5,110 @@
...
+			try {
+				String siteUrl = req.getContextPath() + "/site/" + parts[2] + "/";
 				String queryString = req.getQueryString();
-				if (queryString != null)
-				{
+				if (StringUtils.isNotBlank(queryString)) {
 					siteUrl = siteUrl + "?" + queryString;
 				}
+
+				activeSite.getPages().stream() // get all pages in site
```
Comment (buckett) [inline portal/portal-impl/impl/src/java/org/sakaiproject/portal/charon/handlers/RoleSwitchHandler.java]: Oooo Java 8 goodness. Nice

**Comment by buckett** [comment]
The portal handlers generally use bracket on newline (I hate it), could we try to keep the project consistent, rather than reformatting it to the better style?

**Comment by ern** [comment]
Yeah I am with you but the mixing and spaced tab issues caused enough issues that I was like well lets fix up.

**Comment by ottenhoff** [comment]
This is a super important Sakai 11 fix.... ready for merge or waiting for changes?

**Comment by buckett** [comment]
Yep, good fix, but can we please keep projects consistent.

**Comment by ern** [comment]
@buckett  Switched the braces back to the previous style for consistency!

**Pull request merged**